test/msca/6900/v0.7/storage/ChildConstructorInitializableMock.sol

```diff
@@ -18,15 +18,16 @@
  */
 pragma solidity 0.8.24;
 
-import {IValidationModule} from "@erc6900/reference-implementation/interfaces/IValidationModule.sol";
+import {ConstructorInitializableMock} from "./ConstructorInitializableMock.sol";
 
-interface ISingleSignerValidationModule is IValidationModule {
```

Comment:
I deleted `ISingleSignerValidationModule`, and added `ChildConstructorInitializableMock`, which should be not related actions, but somehow github recognizes this as a rename